### PR TITLE
#2 specify platform when viewing promoCodes

### PR DESCRIPTION
**Pull request by lacostej**
This fixes it for me. Tested for mac and ios. Note that I don't know what happens if you specify the platform and don't really need (e.g. if you only distribute IOS apps)

### CODE REVIEW

File: lib/codes/itunes_connect.rb [hunk 1]
```diff
@@ -21,11 +21,14 @@ def run(args)
         raise "Could not find app using the following information: #{args}. Maybe the app is not in the store. Pass the Apple ID of the app as well!".red
       end
 
+      app=FastlaneCore::ItunesSearchApi.fetch(app_id)
```
Comment (KrauseFx) [inline lib/codes/itunes_connect.rb]: Please add spaces before and after the `=`

File: lib/codes/itunes_connect.rb [hunk 2]
```diff
@@ -21,11 +21,14 @@ def run(args)
...
+      platform=(app['kind']=="mac-software"?"osx":"ios")
```
Comment (KrauseFx) [inline lib/codes/itunes_connect.rb]:
```ruby
platform = (app['kind'] == "mac-software" ? "osx" : "ios")
```

**Comment by KrauseFx** [comment]
Looks great, thanks. Please just update the code style, and it's ready to merge :+1:

**Comment by lacostej** [comment]
Not sure if github notifies if I force push the PR, but I did correct the code style (and sorry for not even thinking about it).

**Comment by KrauseFx** [comment]
Great, thanks :+1:

**Pull request merged**